### PR TITLE
doc: terminology cleanup in enable s5 guide

### PR DESCRIPTION
**Pull request by amyreye**
- Replace SOS or Service OS with Service VM
- Replace UOS or User OS with User VM
- Clean up some of the grammar

Signed-off-by: Amy Reyes <amy.reyes@intel.com>

### CODE REVIEW

File: doc/tutorials/enable_s5.rst [hunk 1]
```diff
@@ -182,8 +183,8 @@ The procedure for enabling S5 is specific to the particular OS:
 #. If the Service VM is being shut down (transitioning to the S5 state), it can call
    ``acrnctl stop vm-name`` to shut down the User VMs.
 
-   .. note:: S5 state is not automatically triggered by a Service VM shutdown; this needs
-      to be run before powering off the Service VM.
+   .. note:: S5 state is not automatically triggered by a Service VM shutdown;
+      this command needs to be run before powering off the Service VM.
```
Comment (fitchbe) [inline doc/tutorials/enable_s5.rst:187]: "this command needs to be run before powering off the Service VM" is one of those subject-object mix-ups due to the passive voice. You can either be explicit about who or what powers off the VM, or you can go with the _total-passivity_ option ("this commend needs to be run before the Service VM is powered off").

**Review by fitchbe — APPROVED** [review]

File: doc/tutorials/enable_s5.rst [hunk 2]
```diff
@@ -22,48 +22,49 @@ S5 Architecture
...
-.. note:: The User VM need to be authorized to be able to request a Shutdown, this is achieved by adding 
-   ``--pm_notify_channel uart,allow_trigger_s5`` in the launch script of that VM.
-   And, there is only one VM in the system can be configured to request a shutdown. If there is a second User 
-   VM launched with ``--pm_notify_channel uart,allow_trigger_s5``, ACRN will stop launching it and throw
-   out below error message:
-   ``initiate a connection on a socket error``
-   ``create socket to connect life-cycle manager failed``
+.. note:: The User VM needs to be authorized to be able to request a shutdown.
+   This is achieved by adding ``--pm_notify_channel uart,allow_trigger_s5`` in
+   the launch script of that VM. And, only one VM in the system can be
```
Comment (dbkinder) [inline doc/tutorials/enable_s5.rst:63]:
```suggestion
   the launch script of that VM. Only one VM in the system can be
```

File: doc/tutorials/enable_s5.rst [hunk 3]
```diff
@@ -216,12 +217,12 @@ How to Test
...
            Main PID: 840 (life_mngr)
 
-   .. note:: For WaaG, we need to close ``windbg`` by using the ``bcdedit /set debug off`` command
+   .. note:: For WaaG, close ``windbg`` by using the ``bcdedit /set debug off`` command
       IF you executed the ``bcdedit /set debug on`` when you set up the WaaG, because it occupies the ``COM2``.
 
 #. Use the ``acrnctl stop`` command on the Service VM to trigger S5 to the User VMs:
 
-   .. code-block:: console
+   .. code-block:: none
```
Comment (dbkinder) [inline doc/tutorials/enable_s5.rst:225]: As @gvancuts would ask, let's get rid of these command prompts.  (Command lines with a ``#`` should use ``sudo``)

File: doc/tutorials/enable_s5.rst [hunk 4]
```diff
@@ -22,48 +22,49 @@ S5 Architecture
...
    :name: s5-architecture
 
-   S5 overall architecture
+   S5 Overall Architecture
 
 - **Scenario I**:
 
     The User VM's serial port device (``ttySn``) is emulated in the
-    Device Model, the channel from the Service VM to the User VM:
+    Device Model. The channel from the Service VM to the User VM is as follows:
```
Comment (dbkinder) [inline doc/tutorials/enable_s5.rst:37]: The text on lines 36-40 is indented too far (two spaces).  Same problem on lines 44-40.

File: doc/tutorials/enable_s5.rst [hunk 5]
```diff
@@ -196,7 +197,7 @@ How to Test
    use of ``vUART1``. We can eliminate the conflict by preventing
    that service from being started
    either automatically or manually, by masking the service
-   using this command
+   using this command:
```
Comment (dbkinder) [inline doc/tutorials/enable_s5.rst:200]: Can you also fix the indenting here too.

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Review by gvancuts — APPROVED** [review]
LGTM. Beware that big changes are coming for the Lifecylce Manager (see #6789) and this document will have to be largely re-written.

**Comment by dbkinder** [comment]
@amyreye PR has merge conflicts that need fixing :(

**Comment by amyreye** [comment]
> @amyreye PR has merge conflicts that need fixing :(

Since the edits on master were substantial and addressed the SOS/UOS terminology issue, I will close this PR. I will then open a new one to address the grammatical and formatting issues that were brought up in the comments.

**Pull request closed without merging**